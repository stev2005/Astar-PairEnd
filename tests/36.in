TTTTGTGTAAGGCTGAACTCCCTCAGTGACGTAGGTCAGCCATGACGAGGCATCCATCGTGCGTTCGACACGACCAGTCCCGCCTGCTTTGGACCTGTTTTCGTGTAAGCCATGCTCGGACAGGCTCTGCTCGCCAACAGGGGCATACCG
15
TTACAAGAGTTACACCAGGGAACCAAGGGA
CTAGAAAATATTCTATTGCCGCGTCTCTCGTG
AACAACTAAGGTGCGAGAGTGGACTAGCTATTAAACGTCGGTGAAACGATCTCGACGTGCTCCAGCCAAGGGGATGCACCTGACACGGAAATGCTAGAAGT
CCTATA
ACTTGATTGGTTGGCTGCCGCTCTAGGAGGAGAACGACCTTAGCGTAAACTCAAACTTCCCGTCGCTGGACCATGT
GTGTGTTTAAGTGTAAGGGTCACGGGGCACAGCGCTGCGGCACTTCTGTCCCGTTACCCGTCAAAGTGTGAAGCAGGTAGCCTTAGTGTAAGGAGGGGACAA
GCGATC
CGGACGGAAGCAATCTCTCCCCCTTGCACGATCGTGCGTCAAGATTTAGACTCGGCATCCGCATAACCGAGGAAGTTGTGGACAGACGTGACTAATAGAGGTATTTGT
GCTGGTCGATGGTTGAACAGGTACTTAGGTTAATGTGTCTTTCGTTGTCTCTGCACCAATATTAGCTCCCAAACGTCCGGATCGCCTGCTCCTAGCCCGGGGGTACGCGCTGAAACCTTGTAAGAGTGTGAGATCAA
CGAAACAAGCTGAGGAATGACCGCTGAATAAAGACGCCGATGGTAATCTCCAGTGGCGGATGCCTGAATGAGAGCCTAGGCTTAGCGTTAAGTCAGTAGGGTCTCGAATGTACGATTGG
TGCGTCAGCGCTCGGTAGTTAGACAAATTTGGCTACAATCGAAAGTTTCGGGCGTCTACGTTAAGACGAC
GCTGAGCTAACGCACGTCTCAGCCACGGCTCTCCGCTTATTCCACTTAAGCCCGGCTAAAACACGGGGCGCCTTCCGACGTTTACCCCGCCGTCTCTCTCTCGGATTGACA
AATCCAGTGACCCAGCCGGATATTTGCTTCGTCCCTGTGATACCATGGCAGAACTTAATTGCTTTAGCTCCTCGACCGTTGGTGTTCTTATCGGCCGTGCATACCATGGGCAACGAAAT
AGCGAGTGCCAGAATCCGATCCACGGCGGCGTTTCACCGGGGATTAAAGATTGTCAGGTATCTTACTCGCTAAATTAATGAGGGGTTACGACTAATCTATATTATTTTTGGTCAGTTCTATTCGTCC
ACTCATCAGCTCTCAACGTATGCGGGACACGAGCCT
