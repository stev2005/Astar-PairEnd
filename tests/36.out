13
15
47
2
33
48
1
50
66
55
31
52
57
62
15
